CHARTER.md

```diff
@@ -0,0 +1,137 @@
+Air, or through any other appropriate means selected by the TSC). The meeting shall be directed by the TSC Chairperson.
+Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC voting member.
+Minutes or an appropriate recording shall be taken and made available to the community through accessible public
+postings.
+
+TSC voting members are expected to regularly participate in TSC meetings and issue triaging. Meeting notes will be released such that absent members of the TSC can provide feedback in any way.
+
+A TSC voting member is automatically converted to a TSC regular member if they do not participate in three consecutive
+TSC votes.
+
+## Section 4. Roles & Responsibilities of the TSC
+
+Subject to such policies as may be set by the CPC, the TSC voting members are responsible for all technical development
+within the webpack project, including:
+
+* Setting release dates.
+* Release quality standards.
+* Technical direction.
+* Project governance and process.
+* GitHub repository management.
+* Conduct and maintain guidelines defined by the OpenJS Foundation Cross Project Council.
+* Maintaining the list of additional collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between collaborators or `webpack` projects.
+
+The TSC voting members will define webpack project's release vehicles.
+
+### Section 4.1. webpack Project Operations
+
+The TSC voting members will establish and maintain a development process for the webpack project. The development
+process will establish guidelines for how the developers and community will operate. It will, for example, establish
+appropriate timelines for TSC review (e.g. agenda items must be published at least a certain number of hours in advance
+of a TSC meeting).
+
+Note that project operations remain subject to any relevant policy or process specified by the OpenJS Foundation board or Cross Project Council.
+
+### Section 4.2. Decision-making, Voting, and/or Elections
+
+#### Section 4.2.1. Elections
+
+These processes are described in webpack's [GOVERNANCE](GOVERNANCE.md) document.
+
+#### Section 4.2.2. Decision Making
+
+For webpack projects' decisions, Collaborators and TSC voting members shall operate under Lazy Consensus ([consensus seeking][]). When an agenda item has appeared to reach a
+consensus, the moderator will ask "Does anyone object?" as a final call for dissent from the consensus. For all votes, a simple majority of all TSC voting members for, or against, the issue wins. A TSC voting member may
+choose to participate in any vote through abstention. Votes are needed to add new members, remove members from the TSC or deciding in project critical issues, such as roadmapping major versions of webpack core.
+
+
+## Section 5. Definitions
+
+* **Contributors**: contribute code or other artifacts, but do not have the right to commit to the code base.
```

Comment:
The term isn't used anywhere in the charter. No point in defining it at all.

Reply:
I guess we can remove it in the opposite direction. Is there a term that would make sense to be added to the Definitions, in your opinion?

Reply:
Not that I think of.